### ISSUE TITLE
(Feature Request) Allow all redirects along with a Black List feature?

### ISSUE THREAD

**Issue by gate1975mlm** [diegocr/CleanLinks#79]
Is there any chance you could add the option to allow all redirects and add a "black List" for sites I only want to block redirects from? I had this feature with another Firefox add on called "redirect cleaner" but that add on was discontinued and so I was so happy to find out about Clean Links. Clean Links is really great but I really miss and need a black List while allowing all redirects from most sites.

Basically I want to stop redirects.as I come across them and add them to a Black List. Having all redirects.blocked by default can and often does break a lot of web sites from working correctly. I saw this minutes after using Clean Links for the first  time when I could not log into my Comcast account which was a redirect. I only want to stop annoying redirects like from "anonymz"

Thanks

**Comment by sumyungguy** [comment]
I'd like to second this request. I only want to enable it for specific websites. I found that it breaks too many websites to have it always on for all websites.
